manifests/init.pp

```diff
@@ -15,7 +15,10 @@
 # $version::                    foreman package version, it's passed to ensure parameter of package resource
 #                               can be set to specific version number, 'latest', 'present' etc.
 #
-# $port::                       Port on which will foreman proxy listen
+# $http_port::                  HTTP port to listen on (if http is enabled)
+#                               type:integer
+#
+# $ssl_port::                   SSL port to listen on (if ssl is enabled)
```

Comment:
Probably but I named it this way because:
  * http boolean goes with http_port setting
  * ssl boolean goes with ssl_port setting

ssl is pre-existing, I'd prefer to 1) keep the convention, and 2) not rename another setting. I *can* rename it but I think most everyone still says SSL when they really mean TLS anyway.  And it's SSL everywhere inside the Foreman UI (settings, manual, etc).